src/cli.rs

```diff
@@ -4,11 +4,10 @@ use resynth::{Error, Loc, Lexer, EOF, Parser, Program};
 use resynth::{warn, error, ok};
 
 use std::path::{Path, PathBuf};
-use std::fs;
-use std::io;
 use std::io::BufRead;
+use std::{fs, io};
 
-use clap::{Arg, App};
+use clap::{Arg, Command, ErrorKind};
```

Comment:
This causes the build to fail: 
```
   Compiling resynth v0.1.7 (/home/scara/src/resynth)
error[E0432]: unresolved import `clap::Command`
  --> src/cli.rs:10:17
   |
10 | use clap::{Arg, Command, ErrorKind};
   |                 ^^^^^^^ no `Command` in the root

For more information about this error, try `rustc --explain E0432`.
error: could not compile `resynth` due to previous error
```

Dependencies in rust are always version locked, so I've no idea how you encountered any deprecation warnings if you didn't also upgrade clap? Did you forget to commit `Cargo.toml` changes upgrading to 3.2.13?

Reply:
I somehow had `clap 3.2.8` installed locally and cargo must have been using it. Since it built for me I completely forgot to update the dependencies in `Cargo.toml`. I'll update the dependency 👍